### ISSUE TITLE
Add Australia supplemental hydro at 10m

### ISSUE THREAD

**Issue by nvkelso** [nvkelso/natural-earth-vector#475]
Per Tom Patterson's new contribution.

**Comment by nvkelso** [comment]
Files have been received, but need processing to put into standard project data property schema, assign min_zooms, add NE IDs, add wikipedia IDs, and localized name variants.

**Comment by nvkelso** [comment]
@ImreSamu do you have time the few weeks to add Wikidata concordances and name localization to new water theme in Natural Earth, please? I'll soon upload the files to GH, waiting on a redelivery.

**Comment by nvkelso** [comment]
Files were re-delivered. Looks to be ~ 500 new features for Australia. There may be additional new features in a couple other layers in North America but that can probably be handled 1 off?

I'll need to process these files to make sure they have `ne_id` unique IDs before we run the Wikidata processes.

**Comment by ImreSamu** [comment]
@nvkelso :

>  to add Wikidata concordances and name localization to new water theme in Natural Earth, 

Yes, 
This issue is **on my to-do list**

I have started a new "test processing"  (  on the  `master` branch   +  latest wikidata dump ) ---->  testing only my data processing pipeline.

> I'll need to process these files to make sure they have ne_id unique IDs before we run the Wikidata processes.

Thanks!

**Comment by nvkelso** [comment]
@ImreSamu I've added https://github.com/nvkelso/natural-earth-vector/pull/580 with the 2 shapefiles (which include `ne_id`). 

Like the other supplemental hydro in Europe and North America, each river segment with a different stroke weight value has it's own feature... so right now they all have different `ne_id`, but after the wikidata_id localization run I'll probably clean that up so the same rivers will have the same ne_id. But ready for you now. Thanks!

**Comment by ImreSamu** [comment]
@nvkelso :

https://docs.google.com/spreadsheets/d/1PtsPGfS-gIojMaPgU2CFCqYQ3O0HCv3CI85B7AeF1Ps/edit#gid=550470213

2 worksheets:
* `river_australia` (301 match  / original:339 =  88% )
*  `lake_australia` (  64 match /  original:84    = 76% ) 

comments:
1.)   IMHO:  "cebuano import" is hard to define now:  because other ( azr/de/..  )  language versions added  ( and English not )

so I have added a `"wd_enwiki"` column ("F")  - contains an English Wikipedia page title if exists. 
So if you don't want these records - you can filter them out by emptiness,

2.)  for a "lake_australia" I have added a `"ne_wd_name_diff"`("G") for checking typo.

* "Lake Goonarrie"  vs. "Lake Goongarrie"
* "Lake Annean" vs.	"Lake Anneen"

3.) No Linestsring geometry for Rivers in wikidata   (only 1..3 Points) - so the extreme distance is normal.

As usual TypeI/II errors are possible.

**Comment by nvkelso** [comment]
Awesome, thank you! I’ll start integrating these today :)

**Comment by nvkelso** [comment]
@ImreSamu FYI in Natural Earth 5 I've added a few languages but didn't communicate that clearly to you.

- Farsi (fa), Hebrew (he), Ukrainian (uk), Urdu (ur); 
- and distinguishing between original Simplified Chinese (zh) and new Traditional Chinese (zht) - there is some special logic I added in https://github.com/nvkelso/natural-earth-vector/blob/master/tools/wikidata/fetch_wikidata.py#L72-L132.

Your Wikidata suggestions look great, thank you! If you can rerun the script to pull in those other languages that'd be stellar to future proof. Else I'll run my update script based on the WikidataID you provided :)

**Comment by nvkelso** [comment]
I added the wikidataid and name_* to https://github.com/nvkelso/natural-earth-vector/pull/580.

**Comment by ImreSamu** [comment]
>  5 I've added a few languages but didn't communicate that clearly to you.

sorry,
* the `Farsi (fa), Hebrew (he), Ukrainian (uk), Urdu (ur)` easy to add to my SQL code
* the ["zh-zht special python logic"](https://github.com/nvkelso/natural-earth-vector/blob/master/tools/wikidata/fetch_wikidata.py#L72-L132) is harder to reimplement in SQL. Maybe I can use PL/Python - but I need to test how this is working 

> Else I'll run my update script based on the WikidataID you provided

for a long time .. probably it would be better to move all language "business logic" to this script.
and not using my `name_*` data 

> If you can rerun the script to pull in those other languages that'd be stellar to future proof.

the last week results were  ~ ( 2% scripts + 98% human matching )  efforts 
so not easy to replicate on my side.

I have manually checked all suggestions (  low score included ) 
so my manual post-processing effort is relatively huge.  ..ouch..

PS.
If you want I can re-check the other hydro data in this calendar week ( without the "zh-zht special python logic"  )
* `lake`
* `lake_europe`
* `river_europe`
* `river_lake_centerlines`

**Comment by nvkelso** [comment]
Agree your scripts are best for the `wikidataid` parts and I'll rely on mine to harvest the localized `names`.

Regarding additional Wikidata ID work, the North America lakes and rivers are _also_ ready to go in the branch associated with https://github.com/nvkelso/natural-earth-vector/pull/580:

- `lake`
- `river_lake_centerlines`
- `lake_europe`
- `river_europe`
- `lake_north_america`
- `river_north_america`

Thanks @ImreSamu!

**Comment by ImreSamu** [comment]
OK.
The `lake_north_america` ; `river_north_america`  is already done: https://github.com/nvkelso/natural-earth-vector/issues/591

**Comment by nvkelso** [comment]
Wonderful! (I missed this issue earlier.)

The NA lakes is no change... and for NA rivers it's just 131 records that gained the IDs (no other changes). I assume just those is limit of additional ask?

**Comment by ImreSamu** [comment]
> and for NA rivers it's just 131 records that gained the IDs (no other changes). I assume just those is limit of additional ask?

**"NA rivers":**   In the coming days, I will re-examine ...  so please no process this yet.  ( but I will try to keep the current matches )

**Comment by ImreSamu** [comment]
@nvkelso 

first part - rivers.

#### "rivers_2021Sep20" : 

5 worksheets

ready to import ( worksheet-name : records )
* river_lake_centerlines :  43 
* river_europe :  9 
* river_north_america :  390 

Not clear match:
* _check_river_lake_centerlines  : 3 
* _check_river_north_america : 32

https://docs.google.com/spreadsheets/d/1MeTjeqLeD91zSOWpxv859QHvicmljVbziWs7Rm0CG8o/edit?usp=sharing

**Comment by nvkelso** [comment]
Thanks @ImreSamu! I'll have  look Monday night.

**Comment by ImreSamu** [comment]
@nvkelso 

second part
#### "lakes_2021Sep20"

4 worksheets

* "lake_north_america" : 78 record
* "lake_europe" : 13 record
* "lake" : 29 record

not clear match : 
* "_check_lake_north_america" :  2 record

https://docs.google.com/spreadsheets/d/1cQ-G41huwM5u0U4S8A7CLpR6eBjWX-YiEZeTK48v4JQ/edit?usp=sharing

**Comment by ImreSamu** [comment]
> The "_check" records were dubious so I skipped them. 

OK,  ( == agree )

> Is it better to have part of the name translated even if it's not 100% the right feature? 
> Let's error on side of caution.

I also prefer quality ( instead of quantity )  ...
